src/iterators.jl

```diff
@@ -71,6 +73,10 @@ function reinit!{dim, N}(ci::CellIterator{dim, N}, i::Int)
         ci.nodes[j] = nodeid
         ci.coords[j] = ci.grid.nodes[nodeid].x
     end
+    if isdefined(ci, :dh) # update celldofs
+        resize!(ci.celldofs, ndofs_per_cell(ci.dh))
```

Comment:
I guess... Do we have any thoughts to not have that. Even with different interpolation of fields there would still be the same number of dofs per element.